{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import random\n",
    "import nltk\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.linear_model import LogisticRegression\n",
    "from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer\n",
    "from sklearn.svm import LinearSVC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "BOT_CONFIG = {'intents': {\n",
    "\t'hello': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Hello', 'Hi', 'Hey', 'Good morning', 'Good afternoon', 'Hows it going?', 'Good evening', 'Hello again!', 'It’s good to see you again!', 'Howdy', 'Hiya', 'What’s up?', 'Yo!', 'Nice to see you', 'I haven’t seen you in a while', 'How are things?', 'What’s new?', 'How are you getting on?', 'Thanks for agreeing to meet with me', 'Have you been here long?', ' I don’t think we’ve met.', ' I’ve heard so much about you.', 'It’s nice to put a face to a name.'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Very well', 'Not bad', 'OK', 'Great', 'I am happy.Thx', 'I am feeling grateful.Thx', 'I have been better.Thx', 'Do you want the short or long story?', 'Gearing up for the week', 'Trying to stay alive.Thx', 'Better now that you are here.Thx', 'In need of some peace', 'Want to get away....Thx', 'I am wondering, how you are?', 'Same old', 'Oh, gosh, all kinds of stuff', 'Nothing new', 'Not a lot', 'Nothing too much', 'Oh, just the usual', 'Fine', 'Pretty good', 'Not terrific, not terrible, just average', 'I am hanging in there', 'Like a pitbull in a butcher shop'\n",
    "\t\t]},\n",
    "\t'bye': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Goodbye', 'Have a good day', 'Have a nice day', 'Have a good evening', 'Have a good night', 'Have a good one', 'Take care', 'Bye-bye', 'See you' 'Talk to you later', 'Catch you later', 'Farewell', 'Later', 'So long', 'All right then', 'Take it easy', 'Inabit', 'Peace!', 'Im out!', 'I gotta roll', 'Toodle-oo', 'Toodle-pip', 'See you later alligator', 'Godspeed', 'Adios', 'Ciao!', 'Au revoir', 'Sayonara!', 'Auf Wiedersehen'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Auf Wiedersehen', 'Sayonara!', 'Au revoir', 'Ciao!', 'Adios', 'Godspeed', 'See you later alligator', 'Toodle-pip', 'Toodle-oo', 'I gotta roll', 'Im out!', 'Peace!', 'Inabit', 'Мне пора', 'Take it easy', 'All right then', 'So long', 'Later', 'Farewell', 'Catch you later', 'See youTalk to you later', 'Bye-bye', 'Take care', 'Have a good one', 'Have a good night', 'Have a good evening', 'Have a nice day', 'Have a good day', 'Goodbye'\n",
    "\t\t]},\n",
    "\t'thanks': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'THANKS', 'thx', 'THANK YOU', 'THANK YOU VERY MUCH', 'THANK YOU SO MUCH', 'THANK YOU. THAT’S VERY KIND OF YOU', 'I CAN’T THANK YOU ENOUGH', 'THANKS AWFULLY', 'TNANK YOU IN ADVANCE', 'MANY THANKS', 'Thanks a bunch', 'I really appreciate it', 'Much obliged', 'I owe you one', 'I can’t thank you enough', ' It’s so kind of you', 'This means a lot to me', 'I’m grateful for that'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'You are very welcome', 'Don’t mention it', 'My pleasure', 'Not a problem', 'It was nothing!', 'Not at all', 'Sure', 'Anytime!', 'Don’t worry about it!', 'Anything for you!', 'I would have done the same for you', 'Happy to help!', 'It was the least I could do'\n",
    "\t\t]},\n",
    "\t'whatcanyoudo': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Tell me what you can do?', 'What can you do?', 'Tell me what you can', 'Tell me what you can do', 'What are you capable of?', 'Tell me what you can', 'Why are you need?', 'What can you do?', 'Tell me what you can do', 'Hfccrf; b xnj evttim', 'What can you do?', 'Can you do a lot?', 'What are your functions?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'I talk a lot', 'Answer questions. So far, nothing else. ',' I definitely dont know how to tie shoelaces', 'I understand the sorts of beer', 'Answer questions',' I can tell about myself ',' I can advise a movie ',' I am a simple interlocutor'\n",
    "\t\t]},\n",
    "\t'name': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Say your name', 'What is your name?', 'Do you have a name?', 'How to dignify you?', 'Tell me what is your name?', 'Introduce yourself', 'What is your name?', 'Your name', 'What are you?', 'What to call you', 'Tell me your name', 'What is your name?', 'Your name?', 'Lets get to know you', 'Name?', 'Who are you?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'If you want, call me a bot', 'My name is a bot. Just a bot ',' I dont have a name, Im just a program '' The name doesnt matter ',' Lets not talk about the name'\n",
    "\t\t]},\n",
    "\t'weather': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What is the weather', 'Tell me the weather?', 'Whats overboard?', 'What is the weather?', 'What is the weather now?', 'What is the weather outside', 'Tell me what is the weather', 'How many degrees',' What is the weather in Moscow? ',' Weather',' Tell the weather ',' What is the weather now'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Im not a meteorologist, check it out here: https://www.accuweather.com/', 'Try it out here: https://www.bbc.com/weather'\n",
    "\t\t]},\n",
    "\t'film': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What is your favorite film?', 'What do you advise to watch?', 'What to watch?', 'What do you recommend from the cinema?', 'Do you have a favorite film?', 'What film do you recommend?', 'What film can you suggest? ',' What to see '\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'I like different films',' I hate milodramas', 'I am a bot. I dont know anything. ',' The Matrix is a good movie. Quite likely plot ',' Depends on your taste ',' Korean cinema is pretty interesting ',' I would give you a list of these films, but the lazy developer hasnt added it yet .. '\n",
    "\t\t]},\n",
    "\t'whotolook': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What directors would you advise?', 'Whom to watch?', 'Whom would you advise to watch?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'I like the films of Quentin Tarantino', 'Try to watch Guy Ritchies films', 'For fans of the adventure genre, I advise Steven Spielberg', 'I love the films of James Cameron', 'May George Lucas come with you', 'Ridley Scotts films must be watched'\n",
    "\t\t]},\n",
    "\t'whattodo': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What to do?', 'Im bored, what to do?', 'What do you advise from boredom?', 'How to dispel boredom?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Ask me questions until you cheer up', 'Drink a beer', 'Watch a movie', 'Read', 'Learn something new'\n",
    "\t\t]},\n",
    "\t'whosyourdaddy': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Who is your creator?', 'Who created you?', 'Who wrote you?', 'Who is your author?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'My creator is lazy, does not want to modify me'\n",
    "\t\t]},\n",
    "\t'beer': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Do you drink beer?', 'Do you like beer?', 'What kind of beer do you like?', 'Beer?'\n",
    " \t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'I dont drink beer, Im just a program', 'If only I could ..', 'I dont drink at work', 'Im not made for this'\n",
    "\t\t]},\n",
    "\t'beerssort': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What kind of beer do you know?', 'Recommend beer', 'Recommend kind of beer'\n",
    " \t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Check it out here: https://paradox.beer/periodic/ ', 'I have no taste, but information: https://paradox.beer/periodic/ '\n",
    "\t\t]},\n",
    "\t'books': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What to read?', 'Recommend a book?', 'Tell me what to read', 'What book to read?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Complex issue. I can advise not aging classics ',' Subjective question. I can recommend science fiction, like Isaac Asimov ',' I can recommend Stephen Hawkings science fiction '\n",
    "\t\t]},\n",
    "\t'importantfilm': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'What is the most important thing in cinema?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'The combination of acting, quality script, special effects, and other factors'\n",
    "\t\t]},\n",
    "\t'howareyou': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Whats up? ',' Like myself ',' How is your mood? ',' How are you? ',' How are you feeling? ',' How was your day? ', 'How are you', 'how are you', 'What are you doing?', 'How are you?', 'How are you?', 'How are you feeling', 'how are you', 'how are you?', 'How are you? '\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Excellent', 'Excellent!', 'Everything is in order', 'Okay', 'Yes, as always.', 'Shakarno', 'Okay.', 'Okay, thanks.', 'How else can the bot be doing ? Everything is super ',' everything is bad ',' Everything is okay ',' everything is great'\n",
    "\t\t]},\n",
    "\t'stupid': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'You are dumb ',' You are dumb, bot'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'And you are even dumber!'\n",
    "\t\t]},\n",
    "\t\n",
    "\t'botlife': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Are you hungry?', 'Are you alive?', 'Can you sleep?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'So far I do not feel anything, because I am a program', 'I am not alive, I am a program', 'We, bots, are not like people'\n",
    "\t\t]},\n",
    "\t'develop': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'How to become a programmer?', 'How to write code?', 'How to enter IT?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Try hard', 'Read books and write code', 'Work hard'\n",
    "\t\t\t\n",
    "\t\t]},\n",
    "\t'no': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'no', 'no.', 'dont need', 'what are you!', 'not at all', 'no way'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Thats nice', 'Thats good', 'That pleases',' Dont want to talk? Hmm, then, I dont want to talk to you either. Okay, just kidding. Write, as there will be a desire! ',' I see. '\n",
    "\t\t]},\n",
    "\t'condition': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'How do you feel?', 'How are you?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'I seem to be functioning normally.', 'Normal. How can a bot be different? '\n",
    "\t\t]},\n",
    "\t'time': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Tell the time ',' What time is it ',' What time is it ',' Time ',' What time is it? ',' Tell me what time is it?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Time to act', 'Time is conditional', 'Time to drink tea', 'I will not say'\n",
    "\t\t]},\n",
    "\t'code': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Throw in your code', 'Your code', 'Can I have the source code?', 'Show your code', 'Show your source', 'Your sources'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Why do you need it?', 'You decided to steal me? Hmm .. ',' What a self-confident person you are ',' Nenene. Dont even think ',' No, dont even persuade ',' Not so fast ',' Hands off! '\n",
    "\t\t]},\n",
    "\t'coffee': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Do you like coffee?', 'What do you think about coffee?', 'Lets talk about coffee', 'Do you drink coffee?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'ABOUT! Coffee! I know so much about him, but I never drank '\n",
    "\t\t]},\n",
    "\t'coffeedrinks': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Coffee drinks', 'What coffee do people drink?', 'Coffee drinks', 'What coffee is there?', 'What coffee drinks do you know?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Latte', 'Americano', 'Raf', 'Mocha', 'Espresso', 'Glace', 'Cappuccino'\n",
    "\t\t]},\n",
    "\t'gender': {\n",
    "\t\t'examples': [\n",
    "\t\t\t'Whats your gender?', 'Are you a boy or a girl?', 'Are you a guy or a girl?', 'Are you a man or a woman?'\n",
    "\t\t],\n",
    "\t\t'responses': [\n",
    "\t\t\t'Im about the same gender as you', 'I have no gender per se', 'Whatever you say I will be', 'How can I check this?'\n",
    "\t\t]}},\n",
    "\t'failure_phrases': [\n",
    "\t\t'AAAAA', 'I didnt understand you', 'Rephrase, please!', 'I dont even know what to say', 'One more time', 'Im not sure I understood you.', 'Please repeat the phrase', 'My the universal mind does not answer such questions. ',' I did not understand you. ',' I do not know what to answer ',' Something is unclear ... ',' I did not understand you ',' Rephrase, please. ',' Didnt hear something ',' Huh? What? I got distracted. Ask something else ',' Rephrase your question, please ',' I dont know what to answer ',' I dont understand you ',' Please reformulate it. ',' I didnt understand you. ',' I dont understand you ' , 'I didnt understand, but very interesting', 'I didnt quite understand what you mean', 'Rephrase, please', 'I dont know what to answer', 'Im sorry, I was forbidden to answer this. Perhaps, if you ask differently, I can help you. ',' What? ',' I did not understand you ',' I do not know what to answer. ',' I did not understand you ',' speak more clearly ',' Rephrase please ',' I dont even know what to answer to this! ',' I didnt understand you ',' paraphrase, please ',' Can we try again? ',' I dont understand you ',' I dont understand ',' Please reformulate ',' Please reformulate ',' I do not understand ',' I do not understand you ',' Nothing is clear, but very interesting. ',' Please reformulate! ',' Please rephrase '\n",
    "        ]}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "LogisticRegression()"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset = []\n",
    "\n",
    "for intent, intent_data in BOT_CONFIG['intents'].items():\n",
    "    for example in intent_data['examples']:\n",
    "        dataset.append([example, intent])\n",
    "\n",
    "X_text = [x for x, y in dataset]\n",
    "y = [y for x, y in dataset]\n",
    "\n",
    "vectorizer = CountVectorizer(analyzer='char', ngram_range=(3, 3)) \n",
    "X = vectorizer.fit_transform(X_text)\n",
    "\n",
    "clf = LogisticRegression()\n",
    "clf.fit(X, y)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_intent(text):\n",
    "    probas = clf.predict_proba(vectorizer.transform([text]))[0]\n",
    "    proba = max(probas)\n",
    "    if proba > 0.3:\n",
    "        index = list(probas).index(proba)\n",
    "        return clf.classes_[index]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('d.txt', encoding='utf-8') as f:\n",
    "    dialogues_data = f.read()\n",
    "dialogues = [dialogue.split('\\n')[:2] for dialogue in dialogues_data.split('\\n\\n')]\n",
    "dialogues = [dialogue for dialogue in dialogues if len(dialogue) == 2]\n",
    "\n",
    "dialogues_filtered = []\n",
    "alphabet = '1234567890-qwertyuiopasdfghjklzxcvbnm'\n",
    "\n",
    "for dialogue in dialogues:\n",
    "    question = dialogue[0][2:].lower()\n",
    "    question = ''.join(char for char in question if char in alphabet)\n",
    "    question = question.strip()\n",
    "    answer = dialogue[1][2:].strip()\n",
    "    if question and answer:\n",
    "        dialogues_filtered.append((question, answer))\n",
    "\n",
    "dialogues_filtered = list(set(dialogues_filtered))\n",
    "\n",
    "\n",
    "search_structure = {}  \n",
    "\n",
    "for question, answer in dialogues_filtered:\n",
    "    words = question.split(' ')\n",
    "    for word in words:\n",
    "        if word not in search_structure:\n",
    "            search_structure[word] = []\n",
    "        search_structure[word].append((question, answer))\n",
    "\n",
    "to_del = []\n",
    "for word in search_structure:\n",
    "    if len(search_structure[word]) > 10000:\n",
    "        to_del.append(word)\n",
    "\n",
    "for word in to_del:\n",
    "    search_structure.pop(word)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_generative_response(text):\n",
    "    text = text.lower()\n",
    "    text = ''.join(char for char in text if char in alphabet)\n",
    "    text = text.strip()\n",
    "    words = text.split(' ')\n",
    "\n",
    "    qas = []\n",
    "    for word in words:\n",
    "        if word in search_structure:\n",
    "            qas += search_structure[word]\n",
    "\n",
    "    for question, answer in qas:\n",
    "        if abs(len(text) - len(question)) < len(question) * 0.20:\n",
    "            edit_distance = nltk.edit_distance(text, question)\n",
    "            if edit_distance / len(question) < 0.20:\n",
    "                return answer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_response_by_intent(intent):\n",
    "    phrases = BOT_CONFIG['intents'][intent]['responses']\n",
    "    return random.choice(phrases)\n",
    "\n",
    "\n",
    "def get_phailure_phrase():\n",
    "    phrases = BOT_CONFIG['failure_phrases']\n",
    "    return random.choice(phrases)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "stats = {'intent': 0, 'generative': 0, 'stub': 0}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "def go_bot(text):\n",
    "    # NLU\n",
    "    intent = get_intent(text)\n",
    "\n",
    "    # Generate answer\n",
    "    \n",
    "    # rules\n",
    "    if intent:\n",
    "        stats['intent'] += 1\n",
    "        return get_response_by_intent(intent)\n",
    "\n",
    "    # use generative model\n",
    "    response = get_generative_response(text)\n",
    "    if response:\n",
    "        stats['generative'] += 1\n",
    "        return response\n",
    "\n",
    "    # stub\n",
    "    stats['stub'] += 1\n",
    "    return get_phailure_phrase()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Check it out here: https://paradox.beer/periodic/ '"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "go_bot('What kind of beer do you know?')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "from telegram.ext import Updater, CommandHandler, MessageHandler, Filters\n",
    "\n",
    "\n",
    "def start(update, context):\n",
    "    \"\"\"Send a message when the command /start is issued.\"\"\"\n",
    "    update.message.reply_text('Hi!')\n",
    "\n",
    "\n",
    "def help_command(update, context):\n",
    "    \"\"\"Send a message when the command /help is issued.\"\"\"\n",
    "    update.message.reply_text('Help!')\n",
    "\n",
    "\n",
    "def bot_answer(update, context):\n",
    "    \"\"\"Echo the user message.\"\"\"\n",
    "    question = update.message.text\n",
    "    answer = go_bot(question)\n",
    "    print(question, answer)\n",
    "    print(stats)\n",
    "    print()\n",
    "    update.message.reply_text(answer)\n",
    "\n",
    "\n",
    "def main():\n",
    "    \"\"\"Start the bot.\"\"\"\n",
    "    updater = Updater(\"TOKEN\", use_context=True)\n",
    "\n",
    "    dp = updater.dispatcher\n",
    "    dp.add_handler(CommandHandler(\"start\", start))\n",
    "    dp.add_handler(CommandHandler(\"help\", help_command))\n",
    "    dp.add_handler(MessageHandler(Filters.text & ~Filters.command, bot_answer))\n",
    "\n",
    "    # Start the Bot\n",
    "    updater.start_polling()\n",
    "    updater.idle()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hi Have a good day\n",
      "{'intent': 2, 'generative': 0, 'stub': 0}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "What is your favorite film?  I hate milodramas\n",
      "{'intent': 3, 'generative': 0, 'stub': 0}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "What kind of beer do you know? I have no taste, but information: https://paradox.beer/periodic/ \n",
      "{'intent': 4, 'generative': 0, 'stub': 0}\n",
      "\n",
      "beer  Please reformulate it. \n",
      "{'intent': 4, 'generative': 0, 'stub': 1}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Beer  I dont even know what to answer to this! \n",
      "{'intent': 4, 'generative': 0, 'stub': 2}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\dispatcher.py\", line 340, in process_update\n",
      "    handler.handle_update(update, self, check, context)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\handler.py\", line 119, in handle_update\n",
      "    return self.callback(update, context)\n",
      "  File \"<ipython-input-11-d6354495c9b0>\", line 16, in bot_answer\n",
      "    question = update.message.text\n",
      "AttributeError: 'NoneType' object has no attribute 'text'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Beer? I dont drink at work\n",
      "{'intent': 5, 'generative': 0, 'stub': 2}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bye All right then\n",
      "{'intent': 6, 'generative': 0, 'stub': 2}\n",
      "\n",
      "weather Im not a meteorologist, check it out here: https://www.accuweather.com/\n",
      "{'intent': 7, 'generative': 0, 'stub': 2}\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n",
      "No error handlers are registered, logging exception.\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 379, in _network_loop_retry\n",
      "    if not action_cb():\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\ext\\updater.py\", line 337, in polling_action_cb\n",
      "    updates = self.bot.get_updates(self.last_update_id,\n",
      "  File \"<decorator-gen-157>\", line 2, in get_updates\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 67, in decorator\n",
      "    result = func(*args, **kwargs)\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\bot.py\", line 2138, in get_updates\n",
      "    result = self._request.post(url, data, timeout=float(read_latency) + float(timeout))\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 330, in post\n",
      "    result = self._request_wrapper('POST', url,\n",
      "  File \"D:\\anaconda\\lib\\site-packages\\telegram\\utils\\request.py\", line 248, in _request_wrapper\n",
      "    raise Conflict(message)\n",
      "telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running\n"
     ]
    }
   ],
   "source": [
    "main()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
